wifiphisher/deauth.py

```diff
@@ -7,8 +7,8 @@
 
 import threading
 import scapy.layers.dot11 as dot11
-from scapy.all import *
-from constants import *
+import scapy.arch.linux as linux
+import wifiphisher.constants as constants
```

Comment:
>Explicit is better than implicit.

When I first started working on the project it took me weeks to figure out which variables were from the `scapy` and which from `wifiphishier` because you used `from scapy impor *`. Being able to know where any variable or function call is coming from is really important for maintaining the project.